pkg/sloop/queries/timerange_test.go

```diff
@@ -55,7 +59,8 @@ func Test_timeRangeTable(t *testing.T) {
 		{"", "abc", "123", true, time.Time{}, time.Time{}},
 		{"", "123", "abc", true, time.Time{}, time.Time{}},
 	}
-	for _, thisRange := range rangeTests {
+	for i, thisRange := range rangeTests {
+		fmt.Print(i)
```

Comment:
why do we need this? can we remove it?

Reply:
It helps especially when test case failed. With the number printed out you can easily target which test is failed. Of course we can remove it does not make sense.

Reply:
Can we just use t.Run arg for that? E.g. we can update the line below to be something like this:

t.Run(fmt.Sprintf("Test id: %v, range: %+v", i, thisRange)

Reply:
updated